app/preview/preview.html

```diff
@@ -1,5 +1,6 @@
 <div class="preview-frame"><iframe ng-src="{{ url }}" name="preview-frame"></iframe></div>
 <p ng-if='file'>
-  Previewing {{ file.title }} <a ng-href="{{ url }}" download="filename">(Download)</a>
+  <span translate>Previewing {{ file.title }}</span>
+  <a ng-href="{{ url }}" download="filename">({{ "Download" | translate }})</a>
```

Comment:
That would be one way to approach it but I've been following a different principle learnt from our experience with AtoM translations where the messages are simplified as much as possible and anything related to visual behavior is handled separately so we have more control on its appearance instead of pushing that responsibility to the translator, e.g. with this approach I could at any point without affecting the message and its translations to remove the outer parenthesis and convert the anchor to a button with the `btn` and `btn-*` CSS classes.
This is just a habit but I'm easy, what do you think? Does it make sense?